sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/StaticSealedClassProtectedMembers.CSharp9.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+record Record
```

Comment:
this should be `final` for the rule to work, right?

Reply:
Yes, it should be `sealed`